src/compiler/proposals-and-stabilization.md

```diff
@@ -0,0 +1,336 @@
+    * An earlier accepted MCP is not a substitute for any later necessary approvals.
+
+[rust-lang/compiler-team]: https://github.com/rust-lang/compiler-team
+[design meeting proposal]: ./meetings.md#steeringplanning-meeting
+
+#### What kinds of comments should go on a MCP in the compiler-team repo?
+Please direct technical conversation to the Zulip stream.
+
+The compiler-team repo issues are intended to be low traffic and used for procedural purposes.
+
+It is recommended that any team member who wishes to "second" a proposal be familiar with the
+relevant code. Anyone can note concerns that shouldn't be overlooked.
+
+#### How does one second an MCP or raise an objection?
+These types of procedural comments can be left on the issue (it's also good to leave a message in
+Zulip). See the previous section. To facilitate a machine parsable scanning of the concerns
+please use the following syntax to formally register a concern:
+
+```
+@rfcbot concern reason-for-concern
+
+<long description of the concern>
+```
+
+And the following syntax to lift a concern when resolved:
+
+```
+@rfcbot resolve reason-for-concern
+```
+
+MCPs can be seconded using:
+
+```
+@rfcbot second
+```
+
+##### Who decides whether a concern is unresolved?
+Usually the experts in the given area will reach a consensus here, but if there is some
+need for a "tie breaker" vote or judgment call, the compiler team leads make the final call.
+
+#### When should MCPs be closed?
+MCPs can be closed:
+
+* by the author, if they have lost interest in pursuing it.
+* by a team lead or expert, if there are strong objections from key members of the team that
+  don't look likely to be overcome.
+* by folks doing triage, if there have been three months of inactivity. In this case, people
+  should feel free to re-open the issue if they would like to "rejuvenate" it.
+
+### What happens if someone makes a contribution that requires an approval and doesn't have one?
+If the approval required for the contribution requires an MCP or an RFC, then the contribution
+should be closed or marked as blocked, with a request to create an MCP or RFC first. If approval of
+a PR is acceptable for the specific contribution (see below), then the approval process can begin.
+
+### Can I work on code experimentally before a approval is gained?
+Of course! You are free to work on PRs or write code. But those PRs should be marked as
+experimental and they should not land, nor should anyone be expected to review them (unless
+folks want to).
+
+## What makes a good proposal?
+A good proposal will address the following:
+
+* **Motivation:** Why is this proposal necessary? What problem does it solve? Why is that problem
+  important?
+* **Design:** What are you proposing?
+* **Implementation notes:** You don't have to talk about the implementation normally, but if there
+  are any key things to note (i.e., it was very invasive to implement), you might note them here.
```

Comment:
Suggestion: maybe also "will this change induce significant maintenance burden" esp. if a proposed change is not used by rustc itself?

Reply:
Fixed!